legal-api/src/legal_api/resources/v2/business/business_filings/business_filings.py

```diff
@@ -540,6 +540,20 @@ def check_authorization(identifier, filing_json: dict,
         # for incorporationApplication and registration, get legalType from nameRequest
         else:
             legal_type = filing_json['filing'][filing_type]['nameRequest'].get('legalType')
+        if (
+            not flags.is_on('supported-change-of-officers-entities')
+            and
+            filing_type == CoreFiling.FilingTypes.CHANGEOFOFFICERS.value
+        ):
+            return (
+                jsonify({
+                    'message': (
+                        f'Officer Filing is currently not available for: '
+                        f'{identifier}.'
+                    )
+                }),
+                HTTPStatus.BAD_REQUEST
+            )
```

Comment:
Create a new function in `PermissionService` and move this, so that we can extend for other filings